src/Altinn.Platform/Altinn.Platform.Events/Events/Services/ProfileService.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Altinn.Common.AccessTokenClient.Services;
+using Altinn.Platform.Events.Configuration;
+using Altinn.Platform.Events.Extensions;
+using Altinn.Platform.Events.Services.Interfaces;
+using Altinn.Platform.Profile.Models;
+using AltinnCore.Authentication.Utils;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Altinn.Platform.Events.Services
+{
+    /// <summary>
+    /// App implementation of the profile service, for app development.
```

Comment:
Comment seems like copy paste from app template?

Reply:
the whole file is imported from app :-) Updated